lib/TopStoriesFeed.jsm

```diff
@@ -492,7 +504,7 @@ this.TopStoriesFeed = class TopStoriesFeed {
 
     if (this.storiesLoaded) {
       updateContent();
-    } else {
+    } else if (this.contentUpdateQueue) {
```

Comment:
I think we should be safer if we just prevent any unintended actions directly from `onAction`. E.g.,

```js
onAction(action) {
  if (discoveryStreamEnabled) {
    switch (action.type) {
      case PREFS_INITIAL_VALUES: …
      case PREF_CHANGED: …
      case UNINIT: …
    }
    return;
  }

  // existing fully running feed stuff
}
```

Reply:
This should also help ensure we don't waste cpu doing work on every NEW_TAB_REHYDRATED or other action

Reply:
Done